canonical-web-and-design/maas-ui
# fix(ui): hide password fields when saving a user

## Done

- When saving a new password this hides the password fields if it is successful.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Click your username in the header.
- Click 'Change password...'
- Enter your current password and new password and click 'Save'.
- When the form has saved the password fields should get hidden.

## Fixes

Fixes: #2921.
